From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
Subject: [PATCH] libavformat: widen smoothstreaming path buffers for snprintf

dirname[] can hold up to 1023 characters; appended suffixes exceed 1024-byte
local path stacks and trigger -Wformat-truncation. Match Fragment storage.
---
 libavformat/smoothstreamingenc.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/libavformat/smoothstreamingenc.c b/libavformat/smoothstreamingenc.c
--- a/libavformat/smoothstreamingenc.c
+++ b/libavformat/smoothstreamingenc.c
@@ -41,8 +41,8 @@ typedef struct Fragment {
     int64_t start_time, duration;
     int n;
     int64_t start_pos, size;
-    char file[1024];
-    char infofile[1024];
+    char file[2048];
+    char infofile[2048];
 } Fragment;
 
 typedef struct OutputStream {
@@ -500,8 +500,8 @@ static int ism_flush(AVFormatContext *s, int final)
     int i, ret = 0;
 
     for (i = 0; i < s->nb_streams; i++) {
         OutputStream *os = &c->streams[i];
-        char filename[1024], target_filename[1024], header_filename[1024], curr_dirname[1024];
+        char filename[2048], target_filename[2048], header_filename[2048], curr_dirname[1024];
         int64_t size;
         int64_t start_ts, duration, moof_size;
         if (!os->packets_written)
